Tasks/Common/azure-arm-rest-v2/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "azure-arm-rest-v2",
+  "version": "2.0.0",
+  "description": "Common Lib for Azure ARM REST apis",
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/Microsoft/azure-pipelines-tasks.git"
+  },
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Microsoft/azure-pipelines-tasks/issues"
+  },
+  "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
+  "dependencies": {
+    "jsonwebtoken": "7.3.0",
```

Comment:
"@types/mocha": "2.2.48",
    "@types/node": "6.0.68",
    "@types/q": "1.0.7",

Reply:
it is done.